pkg/api/service.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	maxRequestedRows int32 = 1000
+	maxRequestedRows     uint32 = 1000
+	maxVectorClockLength int    = 100
```

Comment:
I added this mainly to stop malicious actors from flooding our DB with large requests using huge vector clock queries. We could theoretically make this bigger

Reply:
In practice, this should always be < the number of nodes in the registry. Maybe that can be the upper bound?